config/src/main/java/com/alibaba/nacos/config/server/service/datasource/LocalDataSourceServiceImpl.java

```diff
@@ -249,21 +248,15 @@ public void setHealthStatus(String healthStatus) {
      * @throws Exception Exception.
      */
     private void execute(Connection conn, String sqlFile) throws Exception {
-        Statement stmt = null;
-        try {
+        try (Statement stmt = conn.createStatement()) {
```

Comment:
Is this grammar supported after 1.7 JDK. I remember 1.X version client is 1.6JDK

Reply:
Support JDK 7. This syntax is only used on the server side.